lib/Weathermap/Integrations/Cacti/WeatherMapCactiUserPlugin.php

```diff
@@ -256,9 +256,11 @@ public function handleDefaultView($request, $appObject)
         // get the locale from host app
         $locale = $this->manager->application->getLocale();
 
-        print "<h3>This is the React UI below here</h3>";
+        print "<small>This is the React UI below here</small>";
         print "<h1>INCOMPLETE</h1>";
-        print '<style src="cacti-resources/user/main.css"></style>';
+        print '<style>@import "cacti-resources/user/main.css"</style>';
```

Comment:
also need add `global $config ` and change this line for this:
`        print '<style>@import "' . $config['url_path'] . 'plugins/weathermap/cacti-resources/user/main.css";</style>';`

Reply:
Should be done now.